koku/masu/database/ocp_report_db_accessor.py

```diff
@@ -260,11 +266,7 @@ def _get_reports(self, table, filters=None):
     def get_pod_request_cpu_core_hours(self, start_date, end_date, cluster_id=None):
         """Make a mapping of cpu pod request hours."""
         table = OCPUsageLineItemDailySummary
-        filters = {
-            'data_source': 'Pod',
-            'usage_start__gte': start_date,
-            'usage_start__lte': end_date
-        }
+        filters = create_filter('Pod', start_date, end_date)
```

Comment:
It may be worth add the `cluster_id` [below](https://github.com/project-koku/koku/pull/1720) to the create_filter() method since all of these are also doing `if cluster_id`